shopfloor_base/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,38 @@
+Profiles
+~~~~~~~~
+
+In Inventory / Configuration / Shopfloor / Profiles.
```

Comment:
The base shopfloor_base no more depends on `stock`.  `Inventory` as root into the path still relevant?

Reply:
yeah sorry, I forgot to mention that I still have to cleanup READMEs :sweat_smile: